src/pages/membership/components/membershipInformation/MembershipInformation.tsx

```diff
@@ -11,41 +11,49 @@ import NotificationComponent from '../../../../common/notification/NotificationC
 import {
   RenewMyYouthProfile as RenewMyYouthProfileData,
   RenewMyYouthProfileVariables,
-  MembershipStatus,
-  MembershipDetails,
+  MembershipInformation as MembershipInformationTypes,
 } from '../../../../graphql/generatedTypes';
 import styles from './MembershipInformation.module.css';
 import getFullName from '../../helpers/getFullName';
 import convertDateToLocale from '../../helpers/convertDateToLocale';
 
-const MEMBERSHIP_DETAILS = loader('../../graphql/MembershipDetails.graphql');
+const MEMBERSHIP_INFORMATION = loader(
+  '../../graphql/MembershipInformation.graphql'
+);
 const RENEW_MEMBERSHIP = loader('../../graphql/RenewMyYouthProfile.graphql');
 
-type Props = {
-  expirationDate: string;
-  status: MembershipStatus | undefined | null;
-};
+type Props = {};
 
 function MembershipInformation(props: Props) {
   const [showNotification, setShowNotification] = useState(false);
+  const [successNotification, setSuccessNotification] = useState(false);
   const { t } = useTranslation();
 
-  const { data, loading } = useQuery<MembershipDetails>(MEMBERSHIP_DETAILS, {
-    onError: () => setShowNotification(true),
-  });
+  const { data, loading } = useQuery<MembershipInformationTypes>(
+    MEMBERSHIP_INFORMATION,
+    {
+      onError: () => setShowNotification(true),
+    }
+  );
   const [renewMembership] = useMutation<
     RenewMyYouthProfileData,
     RenewMyYouthProfileVariables
-  >(RENEW_MEMBERSHIP);
+  >(RENEW_MEMBERSHIP, { refetchQueries: ['HasYouthProfile'] });
 
-  const validUntil = convertDateToLocale(props.expirationDate);
+  const validUntil = convertDateToLocale(data?.youthProfile?.expiration);
```

Comment:
> data?.youthProfile?.expiration

Possibly undefined? Will this cause unexpected behavior?

Reply:
If value is undefined function will return empty string.